src/styles/main.scss

```diff
@@ -1,3 +1,88 @@
-body {
-  margin: 0;
+// Colors
+$gray: #eee;
+$pink: #ff69b4;
+$black: #000;
+
+// Sizes
+$cell-size: 100px;
+$border-size: 1px;
+$font-size: 30px;
+$padding-size: 10px;
+
+// Calculation variables
+$calendar-width: (7 * $cell-size) + (8 * $border-size) + (2 * $padding-size);
```

Comment:
1. you shouldn't calculate border if you set `box-sizing: border-box` for all elements, using `*`
2.  padding is unnecessary in your container, you can remove it and simplify your calculation 😉 
3.  remember about adding gaps here too, you can also create variable - example 
`$gap: 1px` - use for `.calendar` and to calculate here `6*$gap`